README.md

```diff
@@ -1 +1,247 @@
-# das-authorization
\ No newline at end of file
+# SFA.DAS.Authorization
+
+This package includes:
+
+* Facade to aggregate multiple authorization concerns into a single call including:
+  * Employer features - Toggling, toggling by user whitelisting, toggling by agreement signing.
+  * Employer roles - User membership checks for an account, user role checks for an account.
+  * Provider permissions - Provider permission checks for an organisation.
+* Cross cutting authorization infrastructure for Mvc and WebAPi.
+* Model binding infrastructure for Mvc and WebApi.
+* Html helper extensions for Mvc.
+
+## Configuration
+
+In addition to the `SFA.DAS.Authorization` package one or more of the following packages should be referenced depending on your application's requirements:
+
+* `SFA.DAS.Authorization.EmployerFeatures`
+* `SFA.DAS.Authorization.EmployerRoles`
+* `SFA.DAS.Authorization.ProviderPermissions`
+
+### StructureMap
+
+The authorization packages include StructureMap registries for wiring up their components:
```

Comment:
We could mention which (if any) are required if `RegisterConcreteTypesAgainstTheFirstInterface()` is used.